ArduCopter/APM_Config.h

```diff
@@ -59,3 +59,5 @@
 //#define USERHOOK_MEDIUMLOOP userhook_MediumLoop();        // for code to be run at 10hz
 //#define USERHOOK_SLOWLOOP userhook_SlowLoop();            // for code to be run at 3.3hz
 //#define USERHOOK_SUPERSLOWLOOP userhook_SuperSlowLoop();  // for code to be run at 1hz
+
+#define DEVO_TELEM_ENABLED ENABLED
```

Comment:
But why? Telemetry stream not works until someone selects "Devo Telemetry" in serial port properties. If default will be 'disabled' then no one will be able to use it without self-compilation